playbooks/aws/openshift-minion/launch.yml

```diff
@@ -65,5 +70,11 @@
       delay: 10
       with_items: ec2.instances
 
-# Apply the configs, seprate so that just the configs can be run by themselves
-- include: config.yml
+- name: "Configure instances"
```

Comment:
The best option I see would be to selectively exclude the include for config.yml if needed.

You can add a tag to the include statement for config.yml (http://docs.ansible.com/playbooks_tags.html)
and then cluster.sh could set a flag when invoking launch that gets propagated to the playbook invocation to add the --skip-tags option when running the launch.yml playbook